src/converse-chatboxviews.js

```diff
@@ -71,7 +71,9 @@ converse.plugins.add('converse-chatboxviews', {
                 /* Override method from backbone.js
                  * If the #conversejs element doesn't exist, create it.
                  */
-                if (!this.el) {
+                if (this.el) {
+                    this.setElement(result(this, 'el'), false);
+                } else {
```

Comment:
Looks like there are changes in this diff that I made. I'm not sure why. I guess you need to rebase onto the latest master, or you worked off of a branch instead of master.